### PR TITLE
MGMT-8532: Infra Env events not available in UI

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-8532

InfraEnv events are now merged with cluster events.

Before:
image

After:
image

### CODE REVIEW

File: src/ocm/components/fetching/fetchEvents.ts [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
-    onSuccess(data);
+    const { data: eventList } = await EventsAPI.list(props);
+
+    if (!props.hostId) {
+      const infraEnvId = await InfraEnvsService.getInfraEnvId(props.clusterId);
+      const { data } = await EventsAPI.list({ infraEnvId: infraEnvId });
+
+      eventList.push(
+        ...data.filter(
+          (event) => !eventList.find((e) => e.eventTime + e.message == event.eventTime + e.message),
```
[the remark this answers is not in the document]
Comment (jkilzi) [reply src/ocm/components/fetching/fetchEvents.ts:20]: JS the ugliest parts...

File: src/ocm/components/fetching/fetchEvents.ts [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
+    const { data: eventList } = await EventsAPI.list(props);
+
+    if (!props.hostId) {
+      const infraEnvId = await InfraEnvsService.getInfraEnvId(props.clusterId);
+      const { data } = await EventsAPI.list({ infraEnvId: infraEnvId });
+
+      eventList.push(
+        ...data.filter(
+          (event) => !eventList.find((e) => e.eventTime + e.message === event.eventTime + e.message),
+        ),
```
Comment (jkilzi) [inline src/ocm/components/fetching/fetchEvents.ts:21]: IIUC there is a chance we get duplicated events???

Comment (jgyselov) [reply src/ocm/components/fetching/fetchEvents.ts]: Between the cluster and infraEnv events, we get more duplicates than unique ones. @jkilzi

Comment (jkilzi) [reply src/ocm/components/fetching/fetchEvents.ts]: @osherdp can you please or somebody from the BE take a look at this part?
We are doing this in order to remove duplicated events coming from the cluster and the infra-env...

Comment (osherdp) [reply src/ocm/components/fetching/fetchEvents.ts]: @slavie maybe can take a look
she knows this part well

Comment (osherdp) [reply src/ocm/components/fetching/fetchEvents.ts]: sorry, mentioned the wrong person
@slaviered

Comment (slaviered) [reply src/ocm/components/fetching/fetchEvents.ts]: The filtering should be done on the BE side. Please open a bug over this issue.

**Comment by jgyselov** [comment]
Resolved by BE, no changes needed in the UI.

**Pull request closed without merging**